### DISCUSSION TITLE
HotKeys. How do I mark selected text as inline code?

### DISCUSSION THREAD

**Discussion by igvspb** [TriliumNext/Trilium#4107] [Q&A] [answered]
Is there any hot key that I can use to mark existing selected text as inline code  (`code`)?

Or is there any autoformatting feature that will allow to do this? If I select text and than press "`" all the selected text will be removed.

**Comment by zadam** [comment]
Not as comfortable, but prepending the existing text with "`" and ending it with "`" works (without selection).

**Reply by meichthys** [reply]
Like @zadam  said but you need to prepend and postpend with a ' (backtick) instead of a double quote. :P

**Comment by igvspb** [comment, marked as answer]
Also I found another workaround. Cut selection (ctrl+x), press "`" (backtick), paste selection (ctrl+v), press "`" (backtick).
